test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/CircularPaginationSample.cs

```diff
@@ -39,13 +39,13 @@ public void Activate()
             ////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
             homeCircular = new CircularPagination();
-            //homeCircular.Position2D = new Position2D(50, 410);
+            homeCircular.Position2D = new Position2D(50, 410);
             homeCircular.Size2D = new Size2D(360, 360);
             homeCircular.BackgroundColor = new Color(0.7f, 0.7f, 0.7f, 0.6f);
 
-            homeCircular.ParentOrigin = ParentOrigin.Center;
+            /*homeCircular.ParentOrigin = ParentOrigin.Center;
```

Comment:
please use //  instead of /**/
thank you.

Reply:
Removed the codes here because they're not necessary.
Done. :)